src/utils/connectionStringUtil.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AuthenticationTypes } from '../models/interfaces';
+
+export function getConnectionString(server: string, authType: string, database: string, userName: string, password: string): string {
+	const serverDatabasePart = `Server=${server};Database=${database};`;
```

Comment:
Since we click on the specific table to create the azure function we have all the necessary information. As a table would have access to the connection info.

Reply:
If we already have an active connection (because we're coming from OE) then why aren't you using the connection string API from STS?

Reply:
Will do this in separate PR. I did not find a connection URI to then pass that into GetConnectiongString Request for STS.